src/pagination.ts

```diff
@@ -141,23 +162,31 @@ export class CursorPaginator<T> {
     });
   }
 
+  private getHasNextAndPrevious(result: { entities: T[]; raw: any[] }, pageSize: number, first: number | undefined, after: string | undefined, before: string | undefined) {
+    const hasMore = result.raw.length > pageSize
+    const showingFirstPage = first !== undefined
```

Comment:
This var declaration is superfluous technically but I added it because I believe it increases readability, making it quicker for engineers to process the code functionality.